lib/ecmascript.mjs

```diff
@@ -2290,7 +2187,7 @@ export const ES = ObjectAssign({}, ES2020, {
   },
   GetIANATimeZonePreviousTransition: (epochNanoseconds, id) => {
     const lowercap = BEFORE_FIRST_DST; // 1847-01-01T00:00:00Z
-    let rightNanos = bigInt(epochNanoseconds).minus(1);
+    let rightNanos = epochNanoseconds;
```

Comment:
Hmm, this was actually a bug that I fixed when I copied over the new polyfill, because I had already fixed it in a pull request on the old polyfill — and this change would revert it back to the old state. Maybe it somehow never got merged in the old repo? I'll investigate this.

Reply:
Ugh, it's in https://github.com/tc39/proposal-temporal/pull/1370 which stalled because of the example being low priority. I had forgotten to split out the bug fix in a separate PR. Maybe we could undo these changes here for now, and then I'll work on a new PR in the proposal repo with the proper test262 tests?

Reply:
Your call. LMK what you'd like to do and I'll make any needed changes later this afternoon. Or you could make them. I'm easy either way.

Reply:
Oh, never mind-- I forgot we agreed in chat that you'd make these changes. Got my PRs mixed up. ;-)

Reply:
I ended up having to copy the commits in order to know exactly what should stay and what should go. It took longer, but at least I know it's correct now.